core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -392,25 +401,25 @@ object ConsoleConsumer extends Logging {
     }
   }
 
-  private[tools] class ConsumerWrapper(topic: Option[String], partitionId: Option[Int], offset: Option[Long], whitelist: Option[String],
+  private[tools] class ConsumerWrapper(topic: Option[String], partitionId: Option[Int], offset: Option[Long], includedList: Option[String],
```

Comment:
nit, since it's not really a list, maybe we should use the name `include` or `includeTopics` instead, similar to what you did here https://github.com/apache/kafka/commit/51796bcdef2e93ee4176415961ab98dd7407ebec#diff-54be3f408ff6ce6c9fb454bd25a3c67b1dea23b5da663f9d7e17b5b946222324R101 ?